tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
+                            'database.\n"Upload dataset" choice will let '
+                            'you choose your own mutation data with the '
+                            'option to load the protein domains from pfam '
+                            'database.',
+                            children=[
+                                html.Div(
+                                    className='needleplot-option-name',
+                                    children='Dataset source:'
+                                ),
+                                dcc.Dropdown(
+                                    id='needle-dataset-select-dropdown',
+                                    options=[
+                                        {
+                                            'label': 'Demo dataset',
+                                            'value': DEMO_KEY
+                                        },
+                                        {
+                                            'label': 'Upload dataset',
+                                            'value': FILE_KEY
+                                        },
+                                        {
+                                            'label': 'UniProt dataset',
+                                            'value': DATABASE_KEY
+                                        },
+                                    ],
+                                    value=DEMO_KEY
+                                )
+                            ]
+                        ),
+                        html.Div(
+                            id='needle-%s-div' % DEMO_KEY,
+                            className='needle-config-data',
                             children=[
                                 html.Div(
-                                    id='needle-dataset-header-div',
+                                    className='needleplot-option-name',
+                                    children='Dataset:'
+                                ),
+                                dcc.Dropdown(
+                                    id='needle-dataset-dropdown',
+                                    options=[
+                                        {
+                                            'label': data['label'],
+                                            'value': i
+                                        }
+                                        for i, data in enumerate(DEMO_DATA)
+                                    ],
+                                    value=0
+                                ),
+                            ]
+                        ),
+
+                        html.Hr(),
+                        html.Div(
+                            id='needle-protein-domains-select-div',
+                            className='needle-config-data',
+                            title='If checked, it will allow the user to load '
```

Comment:
You have to decide between 'you' and 'the user' and it looks like we're going for 'you.'
So I would reword in a much more direct way: "Check to load mutation data ..."

Reply:
Changed in d81a6a0!